README.md

```diff
@@ -265,7 +273,24 @@ The 'channel' layer is an abstraction over transport handling that is suitable f
 
 ### Transport
 
-The 'transport' layer does the heavy lifting of putting & taking bytes off the wire. The interfaces to it are abstract just enough to allow plugging in of different implementations. Transports are modeled as 'Stream' factories. The variation in interface between a server stream and a client stream exists to codify their differing semantics for cancellation and error reporting.
+The Transport layer does the heavy lifting of putting & taking bytes off the
+wire. The interfaces to it are abstract just enough to allow plugging in of
+different implementations. Transports are modeled as `Stream` factories. The
+variation in interface between a server Stream and a client Stream exists to
+codify their differing semantics for cancellation and error reporting.
+
+Note the transport layer API is considered internal to gRPC and has weaker API
+guarantees than the core API under package `io.grpc`.
+
+gRPC comes with two Transport implementations:
```

Comment:
I added it anyway. This is a feature that some people will look for and it's nicer to expose it here.